homework_3.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask
+from sqlalchemy import SQLAlchemy
+from datetime import datetime
+from pydantic import BaseModel
+from datetime import date
+
+app = Flask(__name__)
```

Comment:
Why you created separate file with separate Flask application? You can do you changes in same files as in example